cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -369,6 +369,9 @@ def build_kubeconfig(server):
     # Copy the kubectl binary to the destination directory.
     cmd = ['install', '-v', '/usr/local/bin/kubectl', destination_directory]
     check_call(cmd)
+    # Make the config file readable by the ubuntu user for juju scp
+    cmd = ['chown', 'ubuntu', 'config']
```

Comment:
Actually the reason we should and do use the install command instead of copy is because you can chmod the contents on the same command: 
```
install -v -g mbruzek -o mbruzek mrrobot.txt Downloads/
```